### PR TITLE
Update to use new major versions of prooph components

### PR DESCRIPTION
**Pull request by codeliner**

### CODE REVIEW

File: scripts/add_deadlines.php [hunk 1]
```diff
@@ -0,0 +1,63 @@
...
+            'todo_id' => $todo['id'],
+            'user_id' => $todo['assignee_id'],
+            'deadline' => $nextDay->format(\DateTime::ATOM)
+        ]);
+
+        $commandBus->dispatch($addDeadline);
+    }
+
+    echo "All deadlines successfully added.";
+}
```
Comment (prolic) [inline scripts/add_deadlines.php]: new line at end of file missing

File: scripts/history_replay.php [hunk 2]
```diff
@@ -0,0 +1,94 @@
...
+
+    if (count($failedEvents)) {
+        echo "\033[0;31mSome events didn't make it. Going to retry " . count($failedEvents) . " events\033[0m\n";
+        $retry($failedEvents, 1, $ex, $retry);
+    }
+
+    $replayWatchEvent = $stopWatch->stop('replay');
+
+    echo "Replay done in " . $replayWatchEvent->getDuration() . " ms";
+}
```
Comment (prolic) [inline scripts/history_replay.php]: new line at end of file missing

File: scripts/take_todo_snapshot.php [hunk 3]
```diff
@@ -0,0 +1,81 @@
...
+        $todoId->toString(),
+        $todo,
+        get_todo_version($todo),
+        new \DateTimeImmutable("now", new \DateTimeZone('UTC'))
+    );
+
+    $snapshotStore->save($snapshot);
+
+    echo "Snapshot was taken!\n";
+}
```
Comment (prolic) [inline scripts/take_todo_snapshot.php]: new line at end of file missing

**Comment by codeliner** [comment]
New lines are added

**Pull request merged**